package-lock.json

```diff
@@ -2760,258 +2760,6 @@
         }
       }
     },
```

Comment:
we use yarn so we don't need package-lock.json, delete it, please

Reply:
Sorry, I old habits of using `npm i`. Thanks for pointing ir out :) Done